rust/src/client.rs

```diff
@@ -17,20 +17,27 @@ pub const BUFFER_SIZE: usize = 1024;
 #[derive(Parser, Debug, Encode, Decode)]
 #[command(author, version, about, long_about = None)]
 pub enum ClientCli {
+    /// Query accounts
     #[clap(subcommand)]
     Accounts(Accounts),
+    /// Query blocks
     #[clap(subcommand)]
     Blocks(Blocks),
+    /// Query the chain
     #[clap(subcommand)]
     Chain(Chain),
+    /// Create a checkpoint of the indexer store
```

Comment:
`checkpoints" will eventually (#613) do more (both create and restore). For now, this is probably fine.